daemon/protocol/python/tests/daemon_cli_test.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python
+
+import os
+import re
+import shutil
+import subprocess
+import tempfile
+import time
+import unittest
+
+gopath = os.environ['GOPATH']
+pypath = gopath + '/src/github.com/scootdev/scoot/daemon/protocol/python/scoot'
+cliRef = ['python', pypath + '/scoot.py']
+createS = ['snapshot', 'create']
+checkoutS = ['snapshot', 'checkout']
+runE = ['exec', 'run']
+pollE = ['exec', 'poll']
+
+echo = ['echo']
+
+class TestCliCommands(unittest.TestCase):
+  daemonProcess = None
+  pypath = ''
+     
+  def setUp(self):
+    self.tmpdir = tempfile.mkdtemp()
+
+    # Note: the following does not work - the process from the pool does not have GOPATH defined so it can't find the binary
+    try:
+      self.daemonProcess = subprocess.Popen(['{0}/bin/daemon'.format(gopath), '-execer_type', 'os'])
+    except Exception as e:
+      self.fail('Fail:{0}'.format(e))  
+    started = False
+    start = time.time()
+    elapsedTime = 0
+    cmd = cliRef + echo + ['ping']
```

Comment:
Weird mix of list vars and local definitions (echo vs ['ping']), suggest normalizing these and making var names more descriptive (command_args, echo_args, etc)